.gitignore

```diff
@@ -1,2 +1,3 @@
 *~
 /code-customized.gs
+/out/
```

Comment:
Wouldn't using a more specific name for the JSDoc output folder (like `jsdoc-out`) be more future-proof? I imagine "out" could be the default output folder for many different tools and this would cause conflicts.

Reply:
Yes, easy to change. Was just initially going with the default output-name of the jsdoc tool for the sake of not procrastinating over the name myself. Will change it in a minute.

Reply:
...just to clarify my wording - `...for the sake of not procrastinating over the name myself, if it is better that you decide it anyway...` (wasn't implying that you deciding such things during review is procrastination, rather that I was in danger of getting lost down the rabbit-hole if I focused too deeply on such things at 4 in the morning).